packages/grafana-ui/src/components/ColorPicker/ColorPicker.mdx

```diff
@@ -0,0 +1,12 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+
+<Meta title="MDX|ColorPicker" component={ColorPicker} />
```

Comment:
There is no story for `ColorPicker` at the moment. We should probably create one and make some of the other stories internal.

Reply:
Oh, wait it's under `Pickers`. The stories need to be restructured.

Reply:
Ok, how do we best do that? Just move files around? Should I do it as part of this PR?

Reply:
We can do it in another PR